DeploymentGuide.md

```diff
@@ -4,35 +4,46 @@ Management Server component of uProxy for Orgs (UfO)
 
 ## Deployment
 
-### Create an Instance
+### Deploy an Instance
```

Comment:
Sorry, maybe we can drop this line?  It's kind of repeating the ## Deployment title?